robotsim.py

```diff
@@ -81,6 +82,7 @@ def set_position(self,x,y,w):
 
     def move_forward(self):
         self.movements += 1
+        self.points -= 1
```

Comment:
I would leave this operation at the end, because we can change it to 300-self.movements instead of each time having to decrease the counter; also, the 300 rule only applies if the robot reaches the beginning, so it makes sense to do that at the end of the round.